Formula/libtorrent-rakshasa.rb

```diff
@@ -0,0 +1,42 @@
+class LibtorrentRakshasa < Formula
+  desc "BitTorrent library for rTorrent"
+  homepage "https://github.com/rakshasa/libtorrent"
+  url "https://github.com/rakshasa/rtorrent/releases/download/v0.9.7/libtorrent-0.13.7.tar.gz"
+  sha256 "c738f60f4d7b6879cd2745fb4310bf24c9287219c1fd619706a9d5499ca7ecc1"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "cppunit"
+  depends_on "openssl"
+
+  conflicts_with "libtorrent-rasterbar"
```

Comment:
Please also add the conflict to `libtorrent-rasterbar`.

It also needs an explanation, e.g:
https://github.com/Homebrew/homebrew-core/blob/a7bd0068dde3dfdd465d14ee612d709a7bc9d8e4/Formula/sonarqube.rb#L11